### PR TITLE
EUI-8736 JRD e-Links API V2

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8736

### Change description ###
Changes to EXUI to consume new version of JRD e-Links API

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by Daniel-Lam — DISMISSED** [review]
@johnbenjamin-hmcts - reviewed and all looks good; approved

**Review by DavidJayakumar — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by DavidJayakumar — DISMISSED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@johnbenjamin-hmcts - these changes look fine; approved

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by DavidJayakumar — DISMISSED** [review]
LGTM

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: api/workAllocation/personService.ts [hunk 1]
```diff
@@ -34,8 +36,12 @@ export async function postFindPersonSearch(req: EnhancedRequest, res: Response)
   let searchResult: any = [];
   if (domain === PersonRole.JUDICIAL) {
     try {
-      const headers = setHeaders(req);
       for (const serviceCode of serviceCodes) {
+        // Judicial User search API version to be used depends upon the config entry FEATURE_JRD_E_LINKS_V2_ENABLED's value
+        req.headers.accept = showFeature(FEATURE_JRD_E_LINKS_V2_ENABLED)
```
Comment (udaydenduluri33) [inline api/workAllocation/personService.ts:41]: This can be outside loop. Minor

**Review by udaydenduluri33 — APPROVED** [review]
LGTM

**Pull request merged**